charts/egeria-platform/values.yaml

```diff
@@ -0,0 +1,117 @@
+# Egeria Settings
+egeria:
+  # Global Egeria Settings
+  logging: error
+  version: "3.5"
+  debug: true
+
+  # persistence
+  persistence: true
+  storageClass: ibmc-vpc-block-10iops-tier
+  storageSize: 8Gi
+  repositoryType: "local-graph-repository"
+
+  # communication
+  cohort: mds
+  viewOrg: org
+  kafkaEndpoint: egeria-kafka:9092
+
+  # userid used for admin ops (no security enabled by default)
+  user: garygeeke
+
+  # list of servers that should start on this platform
+  # the names must correspond to a config-<name>.sh script
+  # as well as to the actual instance name
+  serverList: mds1,view1
+
+  # OMAG Server
+  # usually you need to configure a connection to an OMAG server
+  # the values below will result in EGERIA_OMAG_SERVER_NAME and EGERIA_OMAG_SERVER_URL environment variables
+  # if platformUrl is empty this will point to itself
+  omagServer:
+    name: mds1
+    platformUrl: ""
+    
+
+# Additional environment variables that are needed for setup the servers
+extraEnv: {}
+
+# name a secret to mount as environment variables to store secrets
+secretName: ""
+
+replicaCount: 1
+
+image:
+  repository: quay.io/odpi/egeria
+  pullPolicy: Always
```

Comment:
for a 'released' image we prefer IfNotPresent . There is logic in the odpi-egeria-lab chart at least that looks at the version - if it's '-SNAPSHOT' then we do Always. Otherwise we delay start, and load image registry more than needed. Should be possible to override via values if needed

Reply:
I used `Always` because we were using the SNAPSHOT version. Do you want to add the logic here? Or do you want to keep it explicitly?